client/coniksclient/internal/cmd/run.go

```diff
@@ -0,0 +1,193 @@
+package cmd
+
+import (
+	"log"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var help = `
+	- register [name] [key]:
+		Register a new name-to-key binding on the CONIKS-server.
+	- lookup [name]:
+		Lookup the key of some known contact or your own bindings.
+	- exit:
+		Close the REPL and exit the client.
+`
+
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run the test client.",
+	Long:  "Run gives you a REPL, so that you can invoke commands to perform CONIKS operations including registration and key lookup. Currently, it supports:\n" + help,
+	Run: func(cmd *cobra.Command, args []string) {
+		run(cmd)
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(runCmd)
+	runCmd.Flags().StringP("config", "c", "config.toml",
+		"Config file for the client (contains the server's initial public key etc).")
+}
+
+func run(cmd *cobra.Command) {
+	conf := loadConfigOrExit(cmd)
+	cc := p.NewCC(nil, true, conf.SigningPubKey)
+
+	state, err := terminal.MakeRaw(int(os.Stdin.Fd()))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer terminal.Restore(int(os.Stdin.Fd()), state)
+	term := terminal.NewTerminal(os.Stdin, "> ")
+	for {
+		term.SetPrompt("> ")
```

Comment:
Why not set the prompt to something more specific? For example "coniks>" or "coniks-client>" (maybe that's worth debating for 0.2.0, though ;-)